packages/wallet/ui/src/components/ConnectionSettingsDialog.jsx

```diff
@@ -118,7 +118,7 @@ const ConnectionSettingsDialog = ({
             const { value } = e.target;
             setConfigSource(value);
             switch (value) {
-              case 'mainnet':
+              case 'main':
```

Comment:
Oops. Something very unit testable. I'll try to find time to write a regression test.